.theia/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "editor.autoSave": "on"
```

Comment:
Can you check what file this is? I think it got added automatically

Reply:
Done